src/Tickets/Commerce/Gateways/Stripe/Webhooks.php

```diff
@@ -342,10 +495,78 @@ public function get_fields(): array {
 					'data-ajax-action-verify' => 'tec_tickets_commerce_gateway_stripe_verify_webhooks',
 				],
 			],
-			'tickets-commerce-gateway-settings-group-end-webhook'         => [
+			'tickets-commerce-gateway-settings-group-end-webhook' => [
 				'type' => 'html',
 				'html' => '<div class="clear"></div></div>',
 			],
 		];
 	}
+
+	/**
+	 * Get the description for the webhooks.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	protected function get_description_webhook_html(): string {
+		ob_start();
+		$kb_link = sprintf(
+			'<a target="_blank" rel="noopener noreferrer" href="%s">%s</a>',
+			esc_url( 'https://evnt.is/1b3p' ),
+			esc_html__( 'Learn more', 'event-tickets' )
+		);
+		?>
+		<p class="tec-tickets__admin-settings-tickets-commerce-gateway-group-description-stripe-webhooks contained">
+			<?php
+			printf(
+				// Translators: %s A link to the KB article.
+				esc_html__( 'Setting up webhooks will enable you to receive notifications on charge statuses and keep order information up to date for asynchronous payments. %s', 'event-tickets' ),
+				$kb_link // phpcs:ignore StellarWP.XSS.EscapeOutput.OutputNotEscaped, WordPress.Security.EscapeOutput.OutputNotEscaped
+			);
+			?>
+		</p>
+		<?php if ( ! $this->has_valid_signing_secret() ) : ?>
+			<p class="tec-tickets__admin-settings-tickets-commerce-gateway-group-description-stripe-webhooks contained">
+				<?php
+				$url = add_query_arg(
+					[
+						'action'   => self::NONCE_KEY_SETUP,
+						'tc_nonce' => wp_create_nonce( self::NONCE_KEY_SETUP ),
+					],
+					admin_url( '/admin-ajax.php' )
+				);
+
+				$save_link = sprintf(
+					'<a id="tec-tickets__admin-settings-webhook-set-up" data-loading-text="%s" rel="noopener noreferrer" href="%s">%s</a>',
+					esc_attr__( 'Setting up your webhook!', 'event-tickets' ),
+					esc_url( $url ),
+					esc_html_x( 'here', 'Describing where the link is located', 'event-tickets' )
+				);
+				printf(
+					// Translators: %s A link to the automatic webhook setup endpoint.
+					esc_html__( 'We can set up your Webhook automatically! Save your unsaved changes and then just click %s!', 'event-tickets' ),
+					$save_link // phpcs:ignore StellarWP.XSS.EscapeOutput.OutputNotEscaped, WordPress.Security.EscapeOutput.OutputNotEscaped
+				);
+				?>
+			</p>
+		<?php endif; ?>
+		<div class="clear"></div>
+		<?php
+		return ob_get_clean();
+	}
+
+	/**
+	 * Whether we have a valid signing secret.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function has_valid_signing_secret() {
+		$has_signing_key      = tribe_get_option( static::$option_webhooks_signing_key );
+		$is_valid_signing_key = tribe_get_option( static::$option_is_valid_webhooks, false );
+
+		return $has_signing_key && md5( $has_signing_key ) === $is_valid_signing_key;
```

Comment:
If you bail early (above) you can remove the first check - you've already done it:
```suggestion
		return md5( $has_signing_key ) === $is_valid_signing_key;
```